mapdata/WarcraftLegacies/UnitData/Core/uCBD.json

```diff
@@ -0,0 +1,78 @@
+{
+  "OldId": 1868783989,
+  "NewId": 1145193333,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
```

Comment:
If you don't mind could you clean up this Tunnel a little? It shouldn't give any food and shouldn't have an attack. And maybe Fortified armor?

Reply:
yup on it now!

Reply:
i think Techno based it off a burrow, and thats why it has those wierd things